src/drag-and-drop/components/draggable-item-wrapper.tsx

```diff
@@ -27,19 +28,18 @@ export const DraggableItemWrapper: React.FC<IProps> = ({ item, position, draggab
     })
   });
 
-  if (isDragging) {
-    // Hide source element on dragging and render preview only.
-    return null;
-  }
```

Comment:
At Matt's suggestion. To simplify the logic of when to show the preview vs. the DraggableItem, there is now logic in starting in line 34 to show DraggableItemPreview if isDragging, else show the DraggableItem.

Reply:
If I recall, there was a bug when dragging the drop targets where multiple previews were showing simultaneously.  This was a quick suggestion to get it working.  Instead of returning, let this component render both the target and the drag preview (I used this pattern in leafpack).  But on reviewing things I do see that it breaks the pattern used in other components.  I wonder if this would work if put back.